### DISCUSSION TITLE
Version 9.8.0b1

### DISCUSSION THREAD

**Discussion by FilipePS** [FilipePS/Traduzir-paginas-web#644] [Announcements]
<p>This is a beta version, to test it you must install it manually (it will not be updated automatically).</p>
<p>If you use Chrome or Edge you can install by clicking directly on <b>TWP_9.8.0b1_Chromium.crx</b> if you run the new file <b>twp-registry-install.reg</b>.</p>

<ul>
        <li>Added
            <ul>
                <li><b>Improve translation</b> button in page translation panel, with the options:
                    <ul>
                        <li>Select the original website language</li>
                        <li>Select target language</li>
                        <li>Do not sort translation results</li>
                    </ul>
                </li>
                <li><b>Privacy</b> tab on the settings page, with the options:
                    <ul>
                        <li>Select the translation services you want to use</li>
                        <li>Silently use an alternate translation service if the current target language is not supported by the selected translation service</li>
                    </ul>
                </li>
                <li><b>Experimental</b> tab on the settings page, with the options:
                    <ul>
                        <li>Configure your Libretranslate server for text (not page) translation.</li>
                    </ul>
                </li>
            </ul>
        </li>
</ul>

<hr /><em>This discussion was created from the release <a href='https://github.com/FilipePS/Traduzir-paginas-web/releases/tag/v9.8.0b1'>Version 9.8.0b1</a>.</em>

**Comment by yangchen158158** [comment]
Perfect, install and test immediately.
